js/build.js

```diff
@@ -12,8 +12,7 @@ $('[data-login-id]').each(function() {
     sendProcessing: 'Sending...'
   };
   _this.$container = $(this);
-  _this.id = _this.$container.attr('data-login-id');
```

Comment:
@YaroslavOvdii Can you confirm that `_this.id` is not used anywhere in this file?

Reply:
@tonytlwu Yes, `_this.id` do not use anywhere in this file
It was used only to get a `_this.data` from `Fliplet.Widget.getData(_this.id)`